docs/eternalcore/using-API.md

```diff
@@ -0,0 +1,105 @@
+# EternalCore Developer API
+
+As part of open-source community, we support developers who want to create their own plugins using our API. We provide a simple and easy to use API that allows you to create your own plugins using our API.
+
+## 📚 Dependency Management
+
+To use our work in your plugin, You need to install correct artifact for your project. Our plugin supports Maven, Gradle Kotlin, Gradle Groovy and Gradle Groovy and SBT.
+To use latest release check [maven repository](https://repo.eternalcode.pl/#/releases/com/eternalcode/eternalcore-api).
+
+### Add repository:
+
+For Gradle:
+```groovy
+maven {
+    url = uri("https://repo.eternalcode.pl/releases")
+}
+```
+
+For Maven:
+```xml
+<repository>
+    <id>eternalcode-reposilite-releases</id>
+    <name>EternalCode Repository</name>
+    <url>https://repo.eternalcode.pl/releases</url>
+</repository>
+```
+
+### Add dependency:
+
+For Gradle:
+```groovy
+compileOnly("com.eternalcode:eternalcore-api:1.1.0")
+```
+
+For Maven:
+```xml
+<dependency>
+    <groupId>com.eternalcode</groupId>
+    <artifactId>eternalcore-api</artifactId>
+    <version>1.1.0</version>
+    <scope>provided</scope>
+</dependency>
+```
```

Comment:
Same for this, code groups.

Reply:
`compileOnly("com.eternalcode:eternalcore-api:1.1.0")`
this should work for kts and groovy